spec/extensions/fourier_transform_functions.md

```diff
@@ -0,0 +1,506 @@
+# Fourier transform Functions
+
+> Array API specification for Fourier transform functions.
+
+A conforming implementation of the array API standard must provide and support the following functions adhering to the following conventions.
+
+-   Positional parameters must be [positional-only](https://www.python.org/dev/peps/pep-0570/) parameters. Positional-only parameters have no externally-usable name. When a function accepting positional-only parameters is called, positional arguments are mapped to these parameters based solely on their order.
+-   Optional parameters must be [keyword-only](https://www.python.org/dev/peps/pep-3102/) arguments.
+-   Broadcasting semantics must follow the semantics defined in {ref}`broadcasting`.
+-   Unless stated otherwise, functions must support the data types defined in {ref}`data-types`.
+-   Unless stated otherwise, functions must adhere to the type promotion rules defined in {ref}`type-promotion`.
+-   Unless stated otherwise, floating-point operations must adhere to IEEE 754-2019.
```

Comment:
Just a note for a potential future PR: Perhaps we want to move this section elsewhere and just refer to it here, instead of copying and pasting in every extension's page?

Reply:
Oh Ralf was actually ahead of me: https://github.com/data-apis/array-api/pull/189#discussion_r646609536

Reply:
Yes, it's a necessary refactor. I'll open a separate issue for it so we actually do it.